src/gen_experiments/__init__.py

```diff
@@ -598,3 +599,9 @@ def identity(x):
         [this_module],
     ),
 }
+skinny_specs = {
+    "exp3": (
+        ("sim_params.noise_abs", "diff_params.meas_var"),
+        (lambda x: x, lambda x: x)
+    )
+}
```

Comment:
EOF new line

Reply:
Still relevant

Reply:
That's weird, my local files have an EOF line and I've commited and pushed those changes, but those changes are not showing up here in Files changed.

Reply:
check `git config core.autocrlf`.  It should be set to `auto`.  If that's the case and you're still showing this issue, I'll go ahead and merge this anyways.

Reply:
`git config core.autocrlf` returns `true`

Reply:
Ok that's apparently how it's supposed to be in git for windows. :+1: